tests/test_conformer.py

```diff
@@ -9,6 +9,12 @@
     ConformerPositionwiseFeedForwardV1,
     ConformerPositionwiseFeedForwardV1Config,
 )
+from i6_models.parts.conformer.frontend import (
+    ConformerVGGFrontendV1,
+    ConformerVGGFrontendV1Config,
+    ConformerVGGFrontendV2Config,
+    ConformerVGGFrontendV2,
```

Comment:
```suggestion
    ConformerVGGFrontendV2,
    ConformerVGGFrontendV2Config,
```
Just for consistency.